requirements_iati.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+-e git+https://github.com/pwyf/bdd-tester.git#egg=bdd-tester
```

Comment:
Sorry, what I mean is to specify the commit of bdd-tester to use, like we do for flattentool: https://github.com/OpenDataServices/cove/blob/master/requirements.txt#L3